vunit/test/bench.py

```diff
@@ -13,7 +13,7 @@
 import re
 import bisect
 import collections
-from collections import OrderedDict
+from typing import Any, Iterable, List, Union, OrderedDict
```

Comment:
Is there any benefit of importing `OrderedDict` from `typing`? Seems more natural to import it from `collections`.

Reply:
I kind of assumed that its required for the type generic annotations. Just checked and that not true. Changed back.